packages/backend/src/rates/service.ts

```diff
@@ -74,29 +74,13 @@ class RatesServiceImpl implements RatesService {
   }
 
   private async getRates(baseAssetCode: string): Promise<Rates> {
-    const [cachedRate, cachedExpiry] = await Promise.all([
-      this.cachedRates.get(baseAssetCode),
-      this.cachedRates.getKeyExpiry(baseAssetCode)
-    ])
-
-    if (cachedRate && cachedExpiry) {
-      const isCloseToExpiry =
-        cachedExpiry.getTime() <
-        Date.now() + 0.5 * this.deps.exchangeRatesLifetime
-
-      if (isCloseToExpiry) {
-        this.fetchNewRatesAndCache(baseAssetCode) // don't await, just get new rates for later
-      }
```

Comment:
This didn't make sense, using `this.fetchNewRatesAndCache(baseAssetCode)` without `await` just turns it into a sync method, which was still making the request and not "prefetching it".

Now in the the new changes its a simple flow: if the rate is cached, return it, if not, get new rates. No need to check for "isCloseToExpiry"